group_project_v2/utils.py

```diff
@@ -108,15 +103,15 @@ def mean(value_array):
         return None
 
 
-def outsider_disallowed_protected_view(func):
+def user_has_no_access_to_protected_view(func):
```

Comment:
@jbzdak nit: slightly confusing name. The idea behind old name was that this decorator _protects_ a _view_ from unhandled _OutsiderDisallowedError_, which would result in 500 and non-informative message at best.

So, the better name would be `groupwork_access_denied_protected_view`. If it does pass pylint checks - try giving shorted name to exception before supressing :)